src/components/Book.js

```diff
@@ -78,4 +83,10 @@ const Content = styled.div`
   display: flex;
   flex-direction: column;
   justify-content: space-between;
+
+  h3 {
```

Comment:
Personally I'd like having the heading as an individual component rather than having it dependent on a wrapping `div`.